### PR TITLE
Commiting changes for Fuse Integration

### PR DESCRIPTION
**Pull request by KMayuri25**

### CODE REVIEW

File: src/app/partner/click-partner-row/click-partner-row.component.ts [hunk 1]
```diff
@@ -26,7 +27,7 @@ export class ClickPartnerRowComponent implements OnInit {
     this.rowData = params.data;
   }
   onClick() {
-
+console.log(this.rowData.partnerType);
```
Comment (Susmith-roy-incontact) [inline src/app/partner/click-partner-row/click-partner-row.component.ts]: Remove the logger

File: src/app/partner/click-partner-row/click-partner-row.component.ts [hunk 2]
```diff
@@ -26,7 +27,7 @@ export class ClickPartnerRowComponent implements OnInit {
...
     localStorage.setItem('ClickedRowData', JSON.stringify(this.rowData))
```
Comment (Susmith-roy-incontact) [inline src/app/partner/click-partner-row/click-partner-row.component.ts]: Remove the data from local storage when not needed.

File: src/app/partner/create-partner-fuse/create-partner-fuse.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CreatePartnerFuseComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```
Comment (Susmith-roy-incontact) [inline src/app/partner/create-partner-fuse/create-partner-fuse.component.spec.ts]: async

File: src/app/partner/create-partner-fuse/create-partner-fuse.component.ts [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    };
+
+    // this.partnerService.create(this.partnerObject).subscribe((res: Response) => {
+    //   this.router.navigate(['partner']);
+    // }, error => {
+    //   localStorage.setItem('ErrorMsg', error );
+    //   this.showErrorMsg = true;
+    // });
+
+    console.log(this.partnerObject);
```
Comment (Susmith-roy-incontact) [inline src/app/partner/create-partner-fuse/create-partner-fuse.component.ts]: remove logger

File: src/app/partner/create-partner-fuse/create-partner-fuse.component.ts [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+      'name': this.control.partner_name.value,
+      'buId': JSON.stringify(this.auth.getTokenBuNo()),
+      'username': this.control.username.value,
+      'password': this.control.password.value,
+      'config' : (JSON.stringify(this.config_val = {'tenantKey': this.control.tenantKey.value, 'medusa_endpoint': this.control.medusa_endpoint.value, 'warden_authURI': this.control.warden_authURI.value, 'app_token': this.control.app_token.value, 'presenceEndpoint': this.control.presenceEndpoint.value, 'bossAPI': this.control.bossAPI.value})),
+     // 'email' : this.control.email.value,
+      'status' : 'active',
+    };
+
+    // this.partnerService.create(this.partnerObject).subscribe((res: Response) => {
```
Comment (Susmith-roy-incontact) [inline src/app/partner/create-partner-fuse/create-partner-fuse.component.ts]: remove commented code if not needed

File: src/app/partner/update-partner-fuze/update-partner-fuze.component.spec.ts [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+        ClickPartnerRowBuIdComponent,
+        ClickPartnerRowEmailComponent,
+        ClickPartnerRowUsernameComponent,
+        ClickPartnerRowStatusComponent],
+      declarations: [ UpdatePartnerFuzeComponent, ErrorMsgComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
```
Comment (Susmith-roy-incontact) [inline src/app/partner/update-partner-fuze/update-partner-fuze.component.spec.ts]: use async

File: src/app/partner/update-partner-fuze/update-partner-fuze.component.ts [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+    if (this.registerForm.invalid) {
+      return;
+    }
+
+    this.partnerObject.name = this.control.partner_name.value;
+    this.partnerObject.buId = JSON.stringify(this.auth.getTokenBuNo());
+    this.partnerObject.username = this.control.username.value;
+    this.partnerObject.password = this.control.password.value;
+    this.partnerObject.config = (JSON.stringify(this.config_val = {'tenantKey': this.control.tenantKey.value, 'medusa_endpoint': this.control.medusa_endpoint.value, 'warden_authURI': this.control.warden_authURI.value, 'app_token': this.control.app_token.value, 'presenceEndpoint': this.control.presenceEndpoint.value, 'bossAPI': this.control.bossAPI.value}));
+  //  this.partnerObject.email = this.control.email.value;
```
Comment (Susmith-roy-incontact) [inline src/app/partner/update-partner-fuze/update-partner-fuze.component.ts]: remove commented code if not needed

File: src/app/select-format/select-format.component.spec.ts [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+  let fixture: ComponentFixture<SelectFormatComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ SelectFormatComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
```
Comment (Susmith-roy-incontact) [inline src/app/select-format/select-format.component.spec.ts]: use async

File: src/app/select-format/select-format.component.spec.ts [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+    fixture = TestBed.createComponent(SelectFormatComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```
Comment (Susmith-roy-incontact) [inline src/app/select-format/select-format.component.spec.ts]: use async

File: src/app/select-format/select-format.component.ts [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+@Component({
+  selector: 'app-select-format',
+  templateUrl: './select-format.component.html',
+  styleUrls: ['./select-format.component.css']
+})
+export class SelectFormatComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
```
Comment (Susmith-roy-incontact) [inline src/app/select-format/select-format.component.ts]: if not needed remove ngOnInitt

File: src/app/app-routing.module.ts [hunk 11]
```diff
@@ -64,6 +67,11 @@ export const routes: Routes = [
     component: UpdatePartnerComponent,
     canActivate: [AuthGuard]
   },
+  {
+    path: 'partner/update2',
```
Comment (saiprasad1997) [inline src/app/app-routing.module.ts]: give some meaningful name in path

File: src/app/app-routing.module.ts [hunk 12]
```diff
@@ -104,6 +112,16 @@ export const routes: Routes = [
     component: CreatePartnerComponent,
     canActivate: [AuthGuard]
   },
+  {
+    path: 'partner/create2',
```
Comment (saiprasad1997) [inline src/app/app-routing.module.ts]: meaningful path name

File: src/app/partner/create-partner-fuse/create-partner-fuse.component.css [hunk 13]
```diff
@@ -0,0 +1,262 @@
...
+  font-weight: 400;
+  line-height: 22px;
+  font-family: OpenSans;
+}
+.b-dropdown .btn.dropdown-toggle{
+  font-weight: 400;
+  background-color: #fff;
+}
+
+.b-dropdown .btn.dropdown-toggle:hover,
```
Comment (saiprasad1997) [inline src/app/partner/create-partner-fuse/create-partner-fuse.component.css]: add container name

File: src/app/partner/create-partner-fuse/create-partner-fuse.component.ts [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+  };
+
+  ngOnInit() {
+
+    this.registerForm = this.formBuilder.group({
+      username: ['', Validators.required],
+   //   email: ['', [ Validators.email]],
+      password: ['', [Validators.required, Validators.minLength(6)]],
+      business_unit: ['', []],
+    //  warden_authURI: ['', [Validators.required , Validators.pattern(/^(?:([A-Za-z]+):)?(\/{0,3})([0-9.\-A-Za-z]+)(?::(\d+))?(?:\/([^?#]*))?(?:\?([^#]*))?(?:#(.*))?$/)]],
```
Comment (saiprasad1997) [inline src/app/partner/create-partner-fuse/create-partner-fuse.component.ts]: remove commented code

File: src/app/partner/create-partner/create-partner.component.css [hunk 15]
```diff
@@ -107,3 +107,160 @@
   line-height: 22px;
   font-family: OpenSans;
 }
+
+/* dropdowns */
```
Comment (saiprasad1997) [inline src/app/partner/create-partner/create-partner.component.css]: add container code

File: src/app/select-format/select-format.component.css [hunk 16]
```diff
@@ -0,0 +1,262 @@
...
+.b-dropdown .btn.dropdown-toggle:active,
+.b-dropdown .btn.dropdown-toggle.active,
+.b-dropdown .open>.btn.dropdown-toggle {
+  background-color:#fff;
+}
+.b-dropdown .btn.dropdown-toggle:active{
+  box-shadow:none;
+}
+
+.b-dropdown .btn.dropdown-toggle{
```
Comment (saiprasad1997) [inline src/app/select-format/select-format.component.css]: add container name

File: src/app/select-format/select-format.component.css [hunk 17]
```diff
@@ -1,4 +1,4 @@
-.create-partner-data .bottom-container {
```
Comment (saiprasad1997) [inline src/app/select-format/select-format.component.css]: why it is removed

**Pull request merged**